newIDE/app/src/MainFrame/Preferences/PreferencesProvider.js

```diff
@@ -73,8 +73,11 @@ export const getInitialPreferences = () => {
 };
 
 const getPreferences = () => {
-  const preferences =
-    loadPreferencesFromLocalStorage() || getInitialPreferences();
+  const initialPreferences = getInitialPreferences();
+  const localPreferences = loadPreferencesFromLocalStorage();
+  const preferences = localPreferences
+    ? { ...initialPreferences, ...localPreferences } // Merge initial preferences with local preferences in case a new value has been added to the preferences
```

Comment:
Is this really useful? 

It should be done already by loadPreferencesFromLocalStorage:
<img width="777" alt="image" src="https://user-images.githubusercontent.com/1280130/169577804-adab26e4-ff0a-4de7-9d4c-6304514430fd.png">

Reply:
I'll dig into it, because it seemed to me that my local preferences missed a value

Reply:
Indeed, that is not needed!